packages/components/src/components/hds/pagination/nav/arrow.ts

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { assert } from '@ember/debug';
+import {
+  HdsPaginationDirectionValues,
+  HdsPaginationDirectionAriaLabelValues,
```

Comment:
Looking at the failing tests, we'd need three sets of enums for directions, instead of two.
 - one for the `prev`/`next` to match the exposed values for `direction` (currently covered by `HdsPaginationDirectionValues`)
 - one for the visible labels: `Previous` and `Next` (currently not covered)
 - and one for the `ariaLabel` (covered by `HdsPaginationDirectionAriaLabelValues`)